examples/DGmethods/ex_001_dycoms.jl

```diff
@@ -0,0 +1,259 @@
+using MPI
+using CLIMA
+using CLIMA.Mesh.Topologies
+using CLIMA.Mesh.Grids
+using CLIMA.DGmethods
+using CLIMA.DGmethods.NumericalFluxes
+using CLIMA.MPIStateArrays
+using CLIMA.LowStorageRungeKuttaMethod
+using CLIMA.ODESolvers
+using CLIMA.GenericCallbacks
+using CLIMA.Atmos
+using CLIMA.VariableTemplates
+using CLIMA.MoistThermodynamics
+using CLIMA.PlanetParameters
+using LinearAlgebra
+using StaticArrays
+using Logging, Printf, Dates
+using CLIMA.Vtk
+
+@static if haspkg("CuArrays")
+  using CUDAdrv
+  using CUDAnative
+  using CuArrays
+  CuArrays.allowscalar(false)
+  const ArrayType = CuArray 
+else
+  const ArrayType = Array 
+end
+
+if !@isdefined integration_testing
+  const integration_testing =
+    parse(Bool, lowercase(get(ENV,"JULIA_CLIMA_INTEGRATION_TESTING","false")))
+end
+
+
+
+"""
+  Initial Condition for DYCOMS_RF01 LES
+"""
+function Initialise_DYCOMS!(state::Vars, aux::Vars, (x,y,z), t)
+    
+  DF         = eltype(state)
+  xvert::DF  = z
+
+  # Constants as defined in Stevens et. al (2005) 
+  R_d::DF     = 287.0
+  cp_d::DF    = 1015.0
+  cp_v::DF    = 1859.0
+  cp_l::DF    = 4181.0
+  Lv::DF      = 2.47e6
+  epsdv::DF   = 1.61
```

Comment:
Since (from what I understand) these constants are not overriding the `PlanetParameters` used in our`MoistThermodyamics`, I suggest we stick with our constants. Otherwise, we have an initialization that is not consistent with our thermodynamics. (One day we will need an easy way to adjust `PlanetParameters` that does not involve changing the source code, but using our constants is fine for now.)